lib/ansible/modules/windows/win_domain_computer.ps1

```diff
@@ -0,0 +1,218 @@
+#!powershell
+#
+# Copyright 2017, AMTEGA - Xunta de Galicia
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# Ansible "directives"
+# WANT_JSON
+# POWERSHELL_COMMON
+
+# ------------------------------------------------------------------------------
+$ErrorActionPreference = "Stop"
+
+# Preparing result
+$result = @{}
+Set-Attr $result "changed" $false
+
+$debug = @{}
+
+# Parameter ingestion
+$params = Parse-Args $args -supports_check_mode $true
+
+$check_mode    = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -default $false  | ConvertTo-Bool
+$diff_support  = Get-AnsibleParam -obj $params -name "_ansible_diff"       -default $false  | ConvertTo-Bool
+$debug_support = Get-AnsibleParam -obj $params -name "_ansible_debug"      -default $false
+$verbosity     = Get-AnsibleParam -obj $params -name "_ansible_verbosity"  -default $false #  -type "int"
+
+Set-Attr $debug "check_mode"    $check_mode
+Set-Attr $debug "diff_support"  $diff_support
+Set-Attr $debug "debug_support" $debug_support
+Set-Attr $debug "verbosity"     $verbosity
+# Output debug
+If ($debug_support -or $verbosity -gt 0)
```

Comment:
Please don't do this. It may be useful for you when testing, but shouldn't be part of the module.
So the whole debug-option and debug output does not belong in the final module.

Reply:
removed